AppCenterIdentity/AppCenterIdentity/Support/iOS.xcconfig

```diff
@@ -1,5 +1,5 @@
 #include "../../../Config/iOS.xcconfig"
 
-OTHER_LDFLAGS=$(inherited) -framework UIKit
+OTHER_LDFLAGS=$(inherited) -framework UIKit -framework AuthenticationServices
```

Comment:
I wanted to give the following feedback

> We need to add `AuthenticationServices` to the podspec of identity, too.

but then realized that we need to actually add App Center Identity to our podspec, which – I think – should be its own PR.

Reply:
Yeah.. We don't have the podspec for Identity now, it will be added at the time we add Identity to podspec. I'll capture this as a bug.